### PR TITLE
Add camera tests

### PR DESCRIPTION
**Pull request by Adimote**
Will only pass tests once https://github.com/sourcebots/robotd/pull/24 is merged, re-adds the camera tests

### CODE REVIEW

File: robot/camera.py [hunk 1]
```diff
@@ -76,7 +73,7 @@ def _see_to_results(data):
         for token in data["markers"]:
             markers.append(Marker(token))
         # Sort by distance
-        return ResultList(sorted(markers, key=lambda x: x.distance_metres))
+        return ResultList(markers)
```
Comment (RealOrangeOne) [inline robot/camera.py]: I would still argue this list should be sorted by distance. Unless it's handled elsewhere?

File: robot/robot.py [hunk 2]
```diff
@@ -167,7 +174,14 @@ def mode(self):
...
         return self._game.mode
 
-    def __del__(self):
+    def close(self):
         # stop the polling threads
         for camera in self.known_cameras:
             camera._stop_poll()
+        for board_type in self.all_known_boards:
+            for board in board_type:
+                board._clean_up()
```
Comment (RealOrangeOne) [inline robot/robot.py]: I'd suggest renaming `_clean_up` to use the `__del__` method, so it's automatically called when a board is destroyed, and means we're not adding additional custom functionality

File: robot/camera.py (hunk 1 above)
Comment (RealOrangeOne) [inline robot/camera.py]: I would still argue this list should be sorted by distance. Unless it's handled elsewhere?

Comment (Adimote) [reply robot/camera.py]: without support of Polar co-ordinates it's not trivial to sort by distance any more. Feature coming once https://github.com/sourcebots/sb-vision/pull/5 is merged

File: robot/robot.py (hunk 2 above)
Comment (thomasleese) [reply robot/robot.py]: I agree, although probably worth also doing an explicit `del board` too.

**Pull request merged**